python/pyspark/ml/classification.py

```diff
@@ -31,7 +31,8 @@
 
 @inherit_doc
 class LogisticRegression(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, HasMaxIter,
-                         HasRegParam, HasTol, HasProbabilityCol, HasRawPredictionCol):
+                         HasRegParam, HasTol, HasProbabilityCol, HasRawPredictionCol,
+                         HasElasticNetParam, HasFitIntercept, HasStandardization, HasThresholds):
```

Comment:
`HasThresholds` is for multi-class but `ml.LogisticRegression` currently only supports binary classification. Perhaps we should also extract a `HasThreshold` mixin for binary classifier thresholds